frontend/src/App.vue

```diff
@@ -3,9 +3,28 @@
 </template>
 
 <script>
+import firebase from "firebase/app";
+import "firebase/database";
 export default {
   data() {
-    return {};
+    return {
+      quotes: []
+    };
+  },
+  mounted() {
+    this.getQuotes();
+  },
+  methods: {
+    async getQuotes() {
+      await this.$rtdbBind("quotes", firebase.database().ref("quotes"));
+      localStorage.setItem(
+        "quotes",
+        JSON.stringify([
+          "I am closer to you than the air you breathe.",
```

Comment:
Don't duplicate this default quote here, only use ones fetched from the database.